database/migrations/2064_12_25_0000_generate_installed_log.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+
+class GenerateInstalledLog extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		$dateStamp = date('Y/m/d h:i:sa');
+		$message = 'Lychee INSTALLED on ' . $dateStamp;
```

Comment:
If we're only using this to block the installer, maybe we should change "on" to "before" here?

Reply:
Well yes and no.
The file is created by the migration file, but we don't know if the migration has been run:

1. automatically by hand or on update `git pull; php artisan migrate` or via online update.
2. at install `git clone ...; composer install; php artisan migrate`
3. by the installer online https://www.example.com/install.php.

For **1.** then *before* would make sense. For **2.** and  **3.** then *on* makes more sense. :unamused:

Reply:
Fair point :+1: